idl/errors.ml

```diff
@@ -12,6 +12,7 @@
  * GNU Lesser General Public License for more details.
  *)
 open Lwt
+open Log
```

Comment:
Do you need to open Log globally when it seems only to be used in fatal_error_t?